_rules/table-header-cells-refer-to-data-cells.md

```diff
@@ -0,0 +1,245 @@
+---
+name: All table header cells refer to data cells
+
+test_type: atomic
+
+description: |
+  This rule checks that each table header in a data table refers to data cells.
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to `th` elements in a `table`, that has [non-empty](#non-empty) `td` elements, which are [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+### Expectation
+- Each target element with the `headers` attribute refers to other `cells` of the same `table`.
```

Comment:
Wouldn't this expectation subsume #412? I'm having a hard time understanding any difference between this and #412 that would justify the need for both rules.
The expectation includes part of the applicability (the `headers` attribute). If we rewrite the applicability we get: 
This rule applies to `th` elements with a `headers` attribute in a `table` that has [non-empty](#non-empty) `td` elements, which are [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree).
#412 applies to `td` elements in addition
The expectation becomes: 
Each target element refers to other `cells` of the same `table`.
#412 refers only to `th` elements, not all `cells` 
I'm pretty sure we can accommodate these checks in a single rule, don't you think?

Reply:
@carlosapaduarte 

I have updated the rule. This rule only does one thing now.
`Each table header in a data table refers to data cells` & does not confuse the same with `headers` attribute.